web/packages/core/src/ruffle-player.ts

```diff
@@ -772,13 +793,15 @@ export class RufflePlayer extends HTMLElement {
     }
 
     private pointerDown(event: PointerEvent): void {
-        // Give option to disable context menu when touch support is being used
-        // to avoid a long press triggering the context menu. (#1972)
         if (event.pointerType === "touch" || event.pointerType === "pen") {
             this.isTouch = true;
         }
     }
 
+    private contextMenu(): void {
+        this.contextMenuSupported = true;
```

Comment:
Maybe move this to `showContextMenu()`?

Reply:
`showContextMenu()` is not just fired on the contextmenu event. It is also fired in checkLongPress. Though this could be done if switching to nosamu's code from https://github.com/ruffle-rs/ruffle/compare/master...n0samu:ruffle:ios_pointer_event_3, which I actually happen to prefer. Namely, this can be added to the `if (e.type === "contextmenu")`  condition within `showContextMenu()`. However, this is currently a `once` event, and would not be with that change, not that that probably matters.